src/main/kotlin/no/nav/syfo/util/StringUtil.kt

```diff
@@ -0,0 +1,8 @@
+package no.nav.syfo.util
+
+import java.util.*
+
+fun String.lowerCapitalize(): String {
+    return this.lowercase()
+        .replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() }
+}
```

Comment:
Den trenger egentlig ikke å hete `lowerCapitalize`, siden bare `capitalise` betyr stor første bokstav med små bokstaver etter. (Dette er også en erstatning for `capitalize`-metoden som er deprecated).
Jeg synes også denne metoden kanskje er unødig komplisert, men det er vel greit når det er anbefalt av Kotlin-gjengen. https://stackoverflow.com/questions/67553792/why-replace-capitalize-with-a-check-for-whether-it-is-lowercase-before-titlec

Ref en kommentar fra en commit i syfomodiaperson:
```
A capitalised word = Word
An upper case word = WORD
A lower case word = word
A capitalized sentence = Capitalized sentence
```

Reply:
Renavnet til capitalize 👍